csrankings-p.csv

```diff
@@ -207,7 +207,6 @@ Pascal Koiran,Ecole Normale Superieure de Lyon,http://perso.ens-lyon.fr/pascal.k
 Pascal Matsakis,University of Guelph,http://www.cis.uoguelph.ca/~matsakis,NOSCHOLARPAGE
 Pascal Meissner,THWS,https://fiw.thws.de/fakultaet/personen/dozentinnen-und-dozenten/person/prof-dr-ing-pascal-meissner,HkBE3kAAAAAJ
 Pascal Poupart,University of Waterloo,https://cs.uwaterloo.ca/~ppoupart,KhAJWroAAAAJ
-Pascal Schweitzer,TU Kaiserslautern,http://alg.informatik.uni-kl.de/team/schweitzer,NOSCHOLARPAGE

```

Comment:
Removed because Pascal Schweitzer is no longer affiliated with the university and is not part of a computer science department anymore.